<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceBrowserWidget</class>
 <widget class="QWidget" name="ResourceBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>330</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Classes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_baseClassCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KFilterProxySearchLine" name="m_classFilter"/>
       </item>
       <item>
        <widget class="QTreeView" name="m_pimoView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="m_labelResourceViewTitle">
         <property name="text">
          <string>All Resources</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ResourceView" name="m_resourceView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>ResourceView</class>
   <extends>QListView</extends>
   <header>resourceview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
